### PR TITLE
Delete note func

### PR DESCRIPTION
**Pull request by dusariudaykumar**
# Feat: Delete Note Function
## Added :
- created a separated folder for notes API calls
- added delete note 
Screenshot 2022-04-11 010309

### CODE REVIEW

File: src/Pages/HomePage/Homepage.jsx [hunk 1]
```diff
@@ -18,25 +21,12 @@ const Homepage = () => {
 
   const addNoteHandler = async (event, note) => {
```
Comment (irajbhinde) [inline src/Pages/HomePage/Homepage.jsx:22]: You can create a separate file for all the API related functions & then import them here for Homepage.jsx to look cleaner

File: src/Pages/HomePage/Homepage.jsx [hunk 2]
```diff
@@ -50,27 +40,21 @@ const Homepage = () => {
...
-        type: "UPDATE_NOTE",
-        payload: resp.data.notes,
-      });
-      setNotes(initialNotes);
-    } catch (error) {
-      console.log(error.message);
-    }
+    const resp = await editNotesService(editnotes, encodedToken);
+    noteDispatch({
+      type: "UPDATE_NOTE",
```
Comment (irajbhinde) [inline src/Pages/HomePage/Homepage.jsx:45]: I haven't implemented APIs in my macro project yet but after doing few PR reviews, I observed that you can add the dispatch function in your services itself. So just look up if its possible to add this dispatch in your services

**Review by irajbhinde — COMMENTED** [review]
Class naming convention is on point & code is looking clean. Keep up the good work

File: src/services/editNotesService.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+      { note: editnotes },
+      {
+        headers: {
+          authorization: encodedToken,
+        },
+      }
+    );
+    return resp;
+  } catch (error) {
+    console.log(error.message);
```
Comment (sharmayatendra) [inline src/services/editNotesService.js]: you can use toastify to show error on view

**Review by sharmayatendra — COMMENTED** [review]
nothin major caught my eye
code is well written
folder structure is also good

File: src/Pages/HomePage/Homepage.jsx (hunk 2 above)
Comment (dusariudaykumar) [reply src/Pages/HomePage/Homepage.jsx:45]: sure i will do this

File: src/services/editNotesService.js (hunk 3 above)
Comment (dusariudaykumar) [reply src/services/editNotesService.js]: I will add  this

**Pull request merged**